Client/YooChooseNotifier.php

```diff
@@ -200,7 +222,7 @@ protected function getContentTypeId($contentId)
 
         try {
             $contentTypeId = $this->contentService->loadContentInfo($contentId)->contentTypeId;
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
```

Comment:
Not really, we just need to make sure that `contentTypeId` is available, if it's not then we’ll return null.